docs/features/weights.rst

```diff
@@ -0,0 +1,34 @@
+.. _Weights:
+
+*******
+Weights
+*******
+
+This module provides empty weight approximations using mostly empirical textbook methods.
+For now there are only models for small turboprop-sized aircraft, but more may be added in the future.
+Component positions within the aircraft are not considered; all masses are accumulated into a single number.
+
+Single-engine turboprop OEW: ``SingleTurboPropEmptyWeight``
+===========================================================
+
+This model combines estimates from Raymer, Torenbeek, Roskam, and others to compute the total operating empty weight of a small single-engine turboprop aircraft.
```

Comment:
I know these are fairly common aircraft design books, but should they still be cited?  Disregard this reccurring comment if they shouldn't.